### PR TITLE
Issue #291 version 14.0.3 Backware Compatibility

### PR DESCRIPTION
**Pull request by spgarbet**

### CODE REVIEW

**Comment by spgarbet** [comment]
Forgot the test case. Will do later.

**Comment by spgarbet** [comment]
Ready for review.

File: R/redcapDataStructure.R [hunk 1]
```diff
@@ -327,7 +328,7 @@ REDCAP_USER_STRUCTURE <-
              data_quality_execute = character(0),
              api_export = character(0),
              api_import = character(0),
-             api_modules = character(0),
+             api_modules=if(is.null(version) || compareVersion(version, "14.0.3") < 0) NULL else character(0),
```
Comment (nutterb) [inline R/redcapDataStructure.R:331]: need to use `utils::compareVersion` to pass check

Comment (nutterb) [reply R/redcapDataStructure.R]: pushed a commit of my own to fix this

**Review by nutterb — APPROVED** [review]
Passes check and all tests are passing.

**Pull request merged**